web/src/components/tasks/RaffleBingoPaginator.test.tsx

```diff
@@ -0,0 +1,75 @@
+import { RaffleBingoPaginator } from "@/components/tasks/RaffleBingoPaginator";
+import { Task } from "@/lib/types/types";
+import { getTaskStatusUpdatedMessage } from "@/lib/utils/helpers";
+import { generateTask } from "@/test/factories";
+import { useMockRouter } from "@/test/mock/mockRouter";
+import { useMockRoadmap } from "@/test/mock/mockUseRoadmap";
+import { setupStatefulUserDataContext, WithStatefulUserData } from "@/test/mock/withStatefulUserData";
+import { generateUserData } from "@businessnjgovnavigator/shared/test";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+jest.mock("next/router", () => ({ useRouter: jest.fn() }));
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+jest.mock("@/lib/roadmap/buildUserRoadmap", () => ({ buildUserRoadmap: jest.fn() }));
+
+jest.mock("@/lib/async-content-fetchers/fetchTaskByFilename", () => ({
+  fetchTaskByFilename: jest.fn(() =>
+    Promise.resolve({
+      default: "Mocked Task Content",
+    })
+  ),
+}));
+
+describe("<RaffleBingoPaginator />", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+    useMockRouter({});
+    setupStatefulUserDataContext();
+  });
+
+  const renderPaginator = (task: Task): void => {
+    render(
+      <WithStatefulUserData initialUserData={generateUserData({})}>
+        <RaffleBingoPaginator task={task} />
+      </WithStatefulUserData>
+    );
+  };
+
+  it("renders first page without clicking stepper", async () => {
+    const task = generateTask({});
+
+    renderPaginator(task);
+    await waitFor(() => {
+      expect(screen.getByTestId("stepper-0")).toBeInTheDocument();
```

Comment:
I like that in your second test you're checking for the presence of the "Back" button. Because you have logic around that, it would be good to validate that this button isn't present on the tab.